packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -89,14 +88,15 @@ export const Stackbox = () => {
     FREQUENCY_OPTIONS.hour
   );
   const [startDateTime, setStartDateTime] = useState<Date>(
-    new Date(startDateTimeTimestamp)
+    new Date(Date.now())
```

Comment:
Instead of `Date.now()`, let's put it as null.

And then bellow we on `openConfirmStack` we can check if the user didn't change the input. If it didn't change, we set the `Date.now()` there so it will be the time it's submited.

```js
const startDate = startDateTime ? startDateTime.getTime() : Date.now().getTime();
```

Reply:
We need this value so it can be displayed in the date time input. Value is a required prop

Reply:
can we do `Date.now()` there?